packages/react-aria-components/src/Accordion.tsx

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
Just checking, but recommendation for building a mutually exclusive AccordionGroup with arrow key navigation would be something like this?
```
function AccordionGroup() {
  let [openIndex, setOpenIndex] = useState(0);
  let focusManager = useFocusManager({accept: (node) => {
    return node.getAttribute('data-accordion-trigger-g1') === true;
  }});
  return (
    <div onKeyDown={(e) => {
      if (e.key === 'ArrowDown') {
        focusManager.focusNext();
        e.preventDefault();
      } else if (e.key === 'ArrowUp') {
        focusManager.focusPrevious();
        e.preventDefault();
      }
    }}>
      <AccordionItem isOpen={openIndex === 0} onOpenChange={val => val && setOpenIndex(0)}>
        {({isOpen}) => (
          <>
            <Header>
              <Heading level={3}>
                <Button slot="trigger" data-accordion-trigger-g1>{isOpen ? '⬇️' : '➡️'} Item 1</Button>
              </Heading>
            </Header>
            <AccordionPanel>
              <p>Content for panel 1.</p>
            </AccordionPanel>
          </>
        )}
      </AccordionItem>
      <AccordionItem isOpen={openIndex === 1} onOpenChange={val => val && setOpenIndex(1)}>
        {({isOpen}) => (
          <>
            <Header>
              <Heading level={3}>
                <Button slot="trigger" data-accordion-trigger-g1>{isOpen ? '⬇️' : '➡️'} Item 2</Button>
              </Heading>
            </Header>
            <AccordionPanel>
              <p>Panel 2's content.</p>
            </AccordionPanel>
          </>
        )}
      </AccordionItem>
    </div>
  );
}

export const AccordionGroupExample = {
  render: () => <FocusScope><AccordionGroup /></FocusScope>
};
```